__tests__/app.test.js

```diff
@@ -101,3 +101,51 @@ describe("/api/articles/:article_id", () => {
   });
 
  }); // end of "/api/articles/:article_id" testing
+
+ describe("/api/articles", () => {
+
+  test("returns correct status code and full array of 13 articles", () => {
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then((data) => {
+      expect(data.body.articles.length).toBe(13)
+    })
+  })
+
+  test("articles returned are sorted in decending order by created_at property", () => {
+    return request(app)
+    .get("/api/articles")
+    .then((data) => {
+        expect(data.body.articles).toBeSortedBy("created_at", { descending: true });
+    })
+  })
+
+  test("each article object needs specified properties. There should not be a body property", () => {
+      return request(app)
+      .get("/api/articles")
+      .then((data) => {
+          data.body.articles.forEach( (article) => {
+              expect(article.hasOwnProperty('author')).toBe(true);
```

Comment:
This test could be made more thorough by checking the type of the property value.

Reply:
updated!